backend/src/tool_registry.py

```diff
@@ -0,0 +1,15 @@
+import src.tools as tools
+
+toolset = [
+    tools.get_action_items,
+    tools.parse_pdf,
+    tools.get_stock_price,
+    tools.parse_file,
+    tools.parse_webpage,
+    tools.summarize_text,
+]
+
+AGENT_LOOKUP = {
```

Comment:
we have a conflict here, please rebase on latest. These variables moved to a different place in main.